packages/shared/lib/utils/utils.ts

```diff
@@ -128,6 +129,17 @@ export function getBaseUrl() {
     return process.env['NANGO_SERVER_URL'] || localhostUrl;
 }
 
+/**
+ * Get Oauth callback url base url.
+ * @desc for ease of use with APIs that require a secure redirect
+ * redirectmeto is automatically used. This is intentioned
+ * for local development
+ * @see https://github.com/kodie/redirectmeto
+ */
+export function getOauthCallbackUrlBaseUrl() {
```

Comment:
Just a nit, but maybe worth calling this one `getLocalOAuthCallbackUrl` to not confuse with `getGlobalOAuthCallbackUrl`?

Reply:
Agreed. Updated with https://github.com/NangoHQ/nango/pull/758/commits/59f583ef673e701bb72db479fe4bae7a735b92bb